### ISSUE TITLE
[Object Object]

### ISSUE THREAD

**Issue by GopherJ** [fannheyward/coc-rust-analyzer#356]
I'm not sure it's which version caused this issue, but it happens very recently, I noticed it's showing now `[Object Obect]` instead of some details

```
 NORMAL  master  main.rs                                    W9  ⠧ 18.66% Downloading rust-analyzer nightly 2020-08-20 rust-analyzer [object Object]  unix  utf-8  rust   38%   46:41

```

**Comment by GopherJ** [comment]
![image](https://user-images.githubusercontent.com/33961674/90747295-08172c00-e2d1-11ea-93bd-d43d942fe6c4.png)

**Comment by poga** [comment]
I've noticed that the param of the `onNotification` callback has a different shape:

https://github.com/fannheyward/coc-rust-analyzer/blob/master/src/ctx.ts#L49 

```
    client.onNotification(ra.status, async (params) => {
      const status = params.status; // === {"status": "ready"}
      this.statusBar.text = `rust-analyzer ${status}`;
```

However, if I change the line to 
```
      this.statusBar.text = `rust-analyzer ${status.status}`;
```

, Typescript will complain about non-existing property `Property 'status' does not exist on type 'Status'...`.

Maybe somewhere upstream added an additional layer of keys accidentally?

**Comment by GopherJ** [comment]
https://github.com/rust-analyzer/rust-analyzer/commit/1eed036a6e68aee6128d099e3a8f2c06a90b846b

**Comment by fannheyward** [comment]
Fixed with v0.7.15.

**Comment by sharksforarms** [comment]
Thanks for the quick update!
